sgkit/io/plink/plink_reader.py

```diff
@@ -1,18 +1,18 @@
 """PLINK 1.9 reader implementation"""
 from pathlib import Path
-from typing import Any, List, Mapping, Optional, Tuple, Union
+from typing import Any, List, Optional, Tuple, Union
 
 import dask.array as da
 import dask.dataframe as dd
 import numpy as np
 from bed_reader import open_bed
-from dask.array import Array
 from dask.dataframe import DataFrame
 from xarray import Dataset
 
-from sgkit import create_genotype_call_dataset
-from sgkit.model import DIM_SAMPLE
-from sgkit.utils import encode_array
+from ... import create_genotype_call_dataset
+from ...model import DIM_SAMPLE
+from ...utils import encode_array
+from ..utils import dataframe_to_dict
```

Comment:
Nit: Any reason to change these to relative imports? I think the absolute ones are more readable.

Reply:
https://github.com/pystatgen/sgkit/pull/314/commits/77964481f3ced2e9b0d8517f353fef0f1a9e1c27